compiler/rustc_const_eval/src/transform/validate.rs

```diff
@@ -97,6 +118,9 @@ struct TypeChecker<'a, 'tcx> {
     storage_liveness: ResultsCursor<'a, 'tcx, MaybeStorageLive<'static>>,
     place_cache: Vec<PlaceRef<'tcx>>,
     value_cache: Vec<u128>,
+    // If `true`, then the MIR must not contain `UnwindAction::Continue` or
+    // `TerminatorKind::Resume`.
```

Comment:
I think you mixed it up here. If `can_unwind: true`, then `UnwindAction::Continue` is allowed, but not otherwise.

Reply:
Indeed